.travis.yml

```diff
@@ -6,13 +6,28 @@ php:
   - 5.6
   - 7
 
+env:
+   - SYMFONY_VERSION=2.3.*
+   - SYMFONY_VERSION=2.7.*
+   - SYMFONY_VERSION=2.8.*
+   - SYMFONY_VERSION=3.0.*
+
+matrix:
+  exclude:
+    - php: 5.4
+      env: SYMFONY_VERSION=3.0.*
```

Comment:
As [Symfony 2.3 EOL will occurs in may 2017](http://symfony.com/roadmap?version=2.3#checker), i think we should support it.

But i agree to remove this exclusion and removing PHP 5.4 tests (and support)